nubis/terraform/sites/main.tf

```diff
@@ -7,6 +7,18 @@ module "consul" {
   service_name = "${var.service_name}"
 }
 
+module "cloudfront" {
+  source                 = "github.com/nubisproject/nubis-terraform//cloudfront?ref=685e9714ec80afb3c42ee3ce6b7f641b65f33349"
+  enabled                = "${var.enable_cdn}"
+  region                 = "${var.region}"
+  environment            = "${var.environment}"
+  account                = "${var.account}"
+  service_name           = "${var.service_name}"
+  load_balancer_web      = "${module.load_balancer_web.address}"
```

Comment:
I'm not sure if the `load_balancer_web` module output is actually in scope here. I might have to refer to that elsewhere.